easybuild/easyconfigs/m/Mako/Mako-1.0.9-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.9'
```

Comment:
We already have Mako/1.0.7 in the 2018b toolchain, so we should stick to that.

Reply:
@akesandgren But there is no `Mako` with `Python 3`, should I add a `Mako/1.0.7-fosscuda-2018b-Python-3.6.6` easyconfig instead?

Reply:
No there isn't at the moment, but the python version doesn't matter for the single-version-per-toolchain-version rule.
So Mako/1.0.7-fosscuda-2018b-Python-3.6.6 is the way to go here.